concrete/blocks/express_form/controller.php

```diff
@@ -241,6 +243,11 @@ public function action_submit($bID = null)
                         $folder = $filesystem->getFolder($this->addFilesToFolder);
                     }
                     $entityManager->refresh($entry);
+
+                    $notifier = $controller->getNotifier($this);
+                    $notifications = $notifier->getNotificationList();
+                    $notifier->sendNotifications($notifications, $entry, ProcessorInterface::REQUEST_TYPE_ADD);
```

Comment:
Won't this send duplicate notifications? What's this for?

Reply:
the double sends a notification is deleted (this one
is caused by conflict during a merge)